src/main/webapp/app/tasks/task.ts

```diff
@@ -2,6 +2,9 @@
  * A tiny task.
  */
 export interface Task {
-  id: string;
-  name: string;
+  id: string
+  name: string
+  status: string
 }
+
+//here i would prefer using an boolean with: isChecked: true/false - but this attempt worked easier for the beginning
```

Comment:
Totally agree, boolean is way better and I think it might also have made things easier a bit because the sorting get's an issue and assigning the correct status value requires more logic.

Reply:
Yes. After reading your following comment, I definitly would use a boolean next time. My boolean-attempt just caused errors. But after your review I understood much more, so I think I'll understand these errors a way better now.